Dockerfile

```diff
@@ -1,50 +1,44 @@
-# docker build . -t fetchai/fetchd:latest
-# docker run --rm -it fetchai/fetchd:latest /bin/sh
-FROM cosmwasm/go-ext-builder:0.8.2-alpine AS builder
+FROM golang:1.14-buster as builder
```

Comment:
This diff is now confusing. I tried to tell git to do the rename, but it looks like it has ignored me. Basically here are the set of changes:

- `Dockerfile` now renamed to `ref.Dockerfile` this is the same as the parent cosmwasm/wasmd project.
- `hub.Dockerfile` and `gcr.Dockerfile` have been merged together (you need to use the `--target` command to specify the image that you want. This makes the configuration of the docker tag action more straight forward.

Reply:
Cool! lgtm